lib/redis-pool.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const genericPool = require('generic-pool')
```

Comment:
👍 nice one, it's always slightly baffled me that we get this proliferation of service-specific pooling libraries when it's such a repeatable problem...